README.md

```diff
@@ -1,2 +1,46 @@
 # asl-schema
 Sequelize schema for licensing models
+
+* `@asl/schema` provides models for interacting with database objects
+
+## Usage
+
+Install the package:
+
+```bash
+npm install @asl/schema --save-prod
+```
+
+Import as necessary:
+
+```js
+const db = require('@asl/schema');
+const models = db(settings.db);
+```
+
+## Configuration
+
+The service can be configured for local development by setting environment variables in a `.env` file.
+
+The following environment variables are required:
+
+* `DATABASE_NAME` - the name of your postgres database
+
+### Database setup
+
+Scripts for setting up a local database with dev data are available in the [`asl-schema` project](https://github.com/ukhomeoffice/asl-schema). First clone that repo and install the dependencies. Then run the following commands in the schema project directory:
```

Comment:
The wording of this refers to `asl/schema` as if it's a different project rather than the one you're currently looking at.